lib/FusionInventory/Agent/Task/Inventory/MacOS/CPU.pm

```diff
@@ -53,7 +53,8 @@ sub _getCpus {
     }
     close $handle;
 
-    my $type  = $sysprofile_info->{'Processor Name'} ||
+    my $type  = $sysctl_info->{'machdep.cpu.brand_string'} || 
```

Comment:
Value "machdep.cpu.brand_string" It's only for os x. On FreeBSD there is  => hw.model: Pentium(R) Dual-Core  CPU      E5200  @ 2.50GHz.